packages/office-addin-manifest/src/addInTypes.ts

```diff
@@ -50,7 +50,7 @@ export function parseAddInType(value: string): AddInType {
   const addInType = toAddInType(value);
 
   if (!addInType) {
-    throw new ExpectedError(`${value} is not a valid Office add-in type.`);
+    throw new Error(`${value} is not a valid Office add-in type.`);
```

Comment:
@millerds @igor-ribeiiro Let's chat offline. I really think this should be ExpectedError.

Reply:
I feel like this shouldn't be a big deal, that's why I changed to error, because it is the safer approach.

Reply:
@millerds

Reply:
@millerds @akrantz I would like to make it be an expected error for now just to be with convention with the rest of the project. We could change it later in another PR